src/client/game.cpp

```diff
@@ -2717,6 +2731,12 @@ void Game::updatePlayerControl(const CameraOrientation &cam)
 		input->getJoystickSpeed(),
 		input->getJoystickDirection()
 	);
+
+	// Enable sneak lock
+	if (g_settings->getBool("sneak_lock")) {
+		control.sneak = true;
```

Comment:
Thought: What if pressing the sneak key disabled sneak lock?

Reply:
I was thinking about that while testing it, though as I didn't see anyone bring it up in the issue I thought might as well leave it without that. If you think its a good idea I can add it though.

Reply:
If it's simple enough, we both agree this is a good idea, and we can't think of anyone who would consider it a bad idea, let's have it.